felix/fv/bpf_test.go

```diff
@@ -941,6 +941,33 @@ func describeBPFTests(opts ...bpfTestOpt) bool {
 						return out
 					}).Should(ContainSubstring("kernel.unprivileged_bpf_disabled = 1"))
 				})
+
+				It("should cleanup after we disable eBPF", func() {
+					By("Waiting for dp to get setup up")
+
+					ensureAllNodesBPFProgramsAttached(tc.Felixes)
+
+					By("Changing env and restarting felix")
+
+					tc.Felixes[0].SetEvn(map[string]string{"FELIX_BPFENABLED": "false"})
```

Comment:
```suggestion
					tc.Felixes[0].SetEnv(map[string]string{"FELIX_BPFENABLED": "false"})
```
typo in the utility func?

Reply:
Didn't know we had that feature; FWIW, I added the `TestManagesBPF` flag to topology options for the "enable BPF" tests, you could use that (but this is fine too).

Reply:
I am aware of that one, but I am happy to start in ebpf, but then i need to flip it to iptables.